docs/core/tools/index.md

```diff
@@ -91,6 +91,10 @@ The following commands are installed by default:
 
 The CLI adopts an extensibility model that allows you to specify additional tools for your projects. For more information, see the [.NET Core CLI extensibility model](extensibility.md) topic.
 
+## Global commands
+
+For information on global commands such as `aspnet-codegenerator`, see [.NET Core Global Tools overview](global-tools.md).
+
 ## Command structure
 
 CLI command structure consists of [the driver ("dotnet")](#driver), [the command (or "verb")](#command-verb), and possibly command [arguments](#arguments) and [options](#options). You see this pattern in most CLI operations, such as creating a new console app and running it from the command line as the following commands show when executed from a directory named *my_app*:
```

Comment:
```suggestion
CLI command structure consists of [the driver ("dotnet")](#driver), [the command](#command-verb), and possibly command [arguments](#arguments) and [options](#options). You see this pattern in most CLI operations, such as creating a new console app and running it from the command line as the following commands show when executed from a directory named *my_app*:
```

This isn't huge, but it quite often is not a verb. Part of our syntax guidelines is that it is often not a verb (only when it applies to the current project, but that seems like a detail not worth including).